examples/vite-demo-vanilla-bundle/src/examples/example09.ts

```diff
@@ -104,6 +110,11 @@ export default class Example09 {
         hideInFilterHeaderRow: false,
         hideInColumnTitleRow: true
       },
+      // TODO: doesn't work in combo with compoundOperatorList
+      compoundOperatorAltTexts: {
+        // where '=' is any of the `OperatorString` type shown above
+        text: { 'Custom': { operatorAlt: '⊂', descAlt: 'SQL Like' } },
```

Comment:
I would rather use `operatorAlt: '%'`, can you elaborate on the TODO above? Shouldn't the `compoundOperatorList` be enough without the need to also use `compoundOperatorAltTexts`? I'm saying this because I don't fully recall, been too long, if both are needed, all I remember is that `compoundOperatorAltTexts` is strictly for text alternate

Reply:
with compoundOperatorList you cant define your own altText but need to pick from a union string/enum. if you do provide a custom operator list though, the path that applies compoundOperatorAltTexts is skipped

Reply:
hmm ok will have to revisit, we might need a more generic approach to make them both happy

Reply:
so I took a look at the code and I now understand what you meant, but I'm not sure if it's worth doing anything about it. I mean that would require some refactoring and I assume that most user would simply use translated values. What is your taught on this? If it should be changed then it would be better as a separate PR to separate the concerns

Reply:
its somewhat unexpected to behave this way. but yep this is certainly for another PR as it doesnt have anything to do with this topic

Reply:
ok I've opened PR #1541 which should fix this issue

Reply:
I assumed that the PR was ok so I merged it, you can merge master branch to get it. However I would prefer to use something like `{ operatorAlt: '%%', descAlt: 'SQL Like' }` as alternate text, it seems more obvious to me :)

Apart from that, I think this comment is "resolved"